addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -184,6 +247,19 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * @private
+         */
+        _confirmThreadRename() {
```

Comment:
I think "apply" is a better verb than "confirm" in this context.
Confirm is usually more a check or the act of redoing something for the sake of ensuring it's done.

Reply:
indeed